server_src/utils/system-paths.utils.js

```diff
@@ -1,10 +1,14 @@
 const { join } = require("path");
+const { lookpath } = require("lookpath");
 
 const systemRoot = "./";
 const logFolder = "logs";
 
 function getLogsPath() {
   return join(systemRoot, logFolder);
 }
+async function checkIfFunctionExistsInPath(systemCommand) {
+  return await lookpath(systemCommand);
```

Comment:
Isnt this a bit over the top? Calling lookpath directly is clearer for me

Reply:
Possibly, I was just thinking of future but I gather this is the only place we're actually checking paths currently so I can call it. Will amend.

Reply:
Replaced it.